dmcontent/govuk_frontend.py

```diff
@@ -315,10 +315,29 @@ def dm_multiquestion(
     if question.get("question_advice"):
         to_render.append(_question_advice(question))
 
-    to_render += [
-        from_question(q, data, errors, is_page_heading=False)
-        for q in question.questions
-    ]
+    to_skip = set()  # we need to be able to skip some questions, such as followups
+    for q in question.questions:
+        if q.id in to_skip:
```

Comment:
Sorry if I'm misunderstanding the control flow but given we instantiate this as an empty set above, how will this ever be true?

Reply:
Later on in the for loop we add things to the set using `

Reply:
If the syntax isn't clear I can change it to `to_skip.update(...)`

Reply:
I'm still slightly confused about the logic, since this happens first, sorry of I'm being slow, is this it stop it trying to add things more than once?

To be honest I prefer `.update` as symbol based operators are generally more difficult to google.

Reply:
No worries, it's good that you're questioning it, it needs to be clear.

Let me expand the comment.

Reply:
Is that any clearer?